BuxOrg/bux-console
# feat(SPV-969): Added error handler component

How to test?

Run previous spv-wallet version where access keys empty list issue presents. Sign in with a xpriv that does not have access keys. Should show error message
![Screenshot 2024-08-06 at 12 03 01 PM](https://github.com/user-attachments/assets/b6c70a7b-a87d-4466-8539-bead2135db48)
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.
